ocs_ci/templates/workloads/smallfile/SmallFile.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
```

Comment:
not a blocking comment but could you please add few words about this benchmark?
what SmallFile do exactly?